experiment/build/generate_cloudbuild.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
```

Comment:
Probably want a simple test for this. can come in next cl.

Reply:
left a TODO